fe/fe-core/src/main/java/org/apache/doris/nereids/properties/ChildOutputPropertyDeriver.java

```diff
@@ -99,6 +100,40 @@ public PhysicalProperties visitPhysicalHashJoin(PhysicalHashJoin<Plan, Plan> has
         return leftOutputProperty;
     }
 
+    @Override
+    public PhysicalProperties visitPhysicalNestedLoopJoin(PhysicalNestedLoopJoin<Plan, Plan> nestedLoopJoin,
```

Comment:
I didn't go through the logic carefully. IIUC, the property deriver framework and logic is under development and not ready to be used. Maybe someone who is expertise in this should take a look.